.sqlfluff

```diff
@@ -35,3 +35,6 @@ extended_capitalisation_policy = upper
 
 [sqlfluff:rules:convention.casting_style]
 preferred_type_casting_style = cast
+
+[sqlfluff:templater:jinja]
+load_macros_from_path = dbt/macros
```

Comment:
As of this PR, we're now using a custom macro in the `aws-athena/` subdirectory, so we need to update the `.sqlfluff` config to let SQLFluff know where the macros are defined. SQLFluff has [builtin support](https://docs.sqlfluff.com/en/stable/configuration.html#builtin-jinja-macro-blocks) for native dbt macros, but custom macros need to be configured using `load_macros_from_path`. (Sidebar: somehow we've gotten away with using custom macros in the `dbt/macros/tests` directory, in particular the `assert_equal` macro we use for unit testing, but I'm not sure how! I spent some time trying to figure it out but gave up after about an hour, since it doesn't seem necessary to figure out.)

Reply:
Fascinating. And also very convenient/fortunate that the stack of tools we've chosen continue to play so nicely together!